android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -108,7 +108,7 @@ constructor(
       .read(SharedPreferenceKey.PRACTITIONER_ID.name, null)
       ?.extractLogicalIdUuid()
   }
-  private var actionParamUpdatableId: ActionParameter? = null
+  private var actionParameterList: List<ActionParameter>? = emptyList()
```

Comment:
Can we rename this, these are action params holding ids of the resources to be updated. Perhaps we can come up with a better name.

Reply:
Okay